sdk/storage/azure-storage-queue/azure/storage/queue/_queue_client.py

```diff
@@ -481,6 +481,61 @@ def send_message( # type: ignore
         except StorageErrorException as error:
             process_storage_error(error)
 
+    @distributed_trace
+    def receive_one_message(self, **kwargs):
```

Comment:
why not `receive_message` ?:-)

Reply:
Haha, I felt that because we have `receive_messages()` already that `receive_one_message()` is more explicit and avoids any potential confusion. But I don't feel so strongly about it, do you feel strongly about renaming it?

Reply:
Unless there's some pythonic reason I'd be consistent with https://github.com/Azure/azure-sdk-for-net/blob/f2b779d67932ca25a91c5c02218acaf7f1c06308/sdk/storage/Azure.Storage.Queues/src/QueueClient.cs#L1865 and https://github.com/Azure/azure-sdk-for-java/blob/6ed232113b920e957dc221ac8fcb2f05ff643b95/sdk/storage/azure-storage-queue/src/main/java/com/azure/storage/queue/QueueClient.java#L454